### ISSUE TITLE
Feature request - A quick way to close direct messages

### ISSUE THREAD

**Issue by Rumbles** [RocketChat/Rocket.Chat.Electron#679]
## My Setup
- Operating System: Fedora
- App Version: 2.10.1

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
If someone sends me a direct message the DM username is left in the menu, I would appreciate an easy way to close the DM.

### Current Behavior
Currently closing a DM involves hovering over the name, clicking the 3 dots, clicking hide the room, confirming. You can't use /leave or /part or use a keyboard shortcut (please correct me if I am wrong)

### Expected Behavior
A quick way to close a DM would be great. Ctrl-D for example isn't a common bind, and is commonly used to disconnect. But honestly, any way to make the closure of DMs quicker would be appreciated. I have a load of rarely used DMs that I want rid of, but the process is too long and I get fed up.

**Comment by TheMythPT** [comment]
Well, hope this helps someone.
on the DM:
```
Open Menu
Choose Room Info
Choose Edit
Choose Delete
```
And done!

**Comment by TheMythPT** [comment]
I know it's a bit harsh, but well.. 🤷‍♂️

**Comment by xfirf** [comment]
I would like to have a shortcut for this too. Its complicated and not that userfriendly when you have to use the mouse / touchpad to close a direct chat.

**Comment by MLefebvreICO** [comment]
I want this too. Tired of having multiple conversations pilling up that I have do manually clean via the `...` which takes time.

If you could have an `X` next to the avatar of the user, to close (without confirmation), it would be nice

**Comment by ahartlaub** [comment]
Yes - the 'don't remind me option' that was added so you don't have to confirm is nice when trying to close, but I get ~80 unique people messaging me on a normal basis.  I don't really want to have all of them show in the list all the time - if it could say 'no new message from [user] in [x] days, hide room' automatically that would be spectacular.

**Comment by piersy** [comment]
Just chiming in.

Auto close of old DMs would be awesome and usingn ctrl+D would be great. I would love to see both features.

Additionally requesting confirmation to hide a room seems like overkill. Because nothing is lost when you hide a room, so in the worst case if a user mistakenly closes a DM, they can just open it again. You wouldn't request confirmation for closing a window on a user's desktop so why request confirmation here?

**Comment by rh-jelabarre** [comment]
> Well, hope this helps someone.
> on the DM:
> 
> ```
> Open Menu
> Choose Room Info
> Choose Edit
> Choose Delete
> ```
> 
> And done!

Just looked at the RC client, and there is no "edit" or "delete" option under "Room Info" for a direct message.  So this doesn't work anyway.

From what I'm reading here, "Hide Room" is the only way to remove a "Direct Message" entry.  If this is in fact the only way to do it, it is very counter-intuitive.  I am looking to close the item, not stick it in the background.  "Channels" can be closed (technically, it's "leave room").  It would seem obvious that direct messages should be handled the same way.

**Comment by Perflyst** [comment]
I also wonder from where @TheMythPT has these kind of menus

**Comment by oddmyth** [comment]
I would add that adding an expand/contract to any or all of the headings in the left bar would be amazing, and I probably wouldn't care so much about hiding DM rooms if I could just get the real estate back.

Off topic but being able to choose the positioning of the headings in the left hand list would also be amazing.

**Comment by Faldal** [comment]
Still not implemented? I wonder what would be so "hard" or "time consuming" to implement such a simple feature, which you can't deny is something that should be "standard".

**Comment by MarkFalto** [comment]
also looking for this feature.

**Comment by GunnerBasil** [comment]
having a specifically outlined "close room" or delete button would be mega helpful. Hope this gets more attention.

**Comment by iburlakov25** [comment]
Hi, what about auto-hiding DMs if you havent been talking to the person for 24/48/72h ? Its very annoying to clear the list when you have 50+ messages every day. Slack for example has this feature.
